pkg/utils/miniojob/minioJob_test.go

```diff
@@ -328,6 +338,335 @@ func TestMCConfigSet(t *testing.T) {
 	}
 }
 
+func TestSupportCallhome(t *testing.T) {
+	mcCommand := "support/callhome"
+	funcs := JobOperation[mcCommand]
+	testCase := []struct {
+		name             string
+		args             map[string]string
+		expectCommand    string
+		expectError      bool
+		expectFileNumber int
```

Comment:
Looks like this field is never used.

Reply:
Copy from another Testing Function. That command don't need any files. @ramondeklein